proto/utxorpc/v1alpha/query/query.proto

```diff
@@ -109,15 +109,34 @@ message ReadDataResponse {
   ChainPoint ledger_tip = 2; // The chain point that represent the ledger current position.
 }
 
+// Request to get a transaction by hash
+message GetTxRequest {
+  bytes hash = 1; // The hash of the transaction.
+  google.protobuf.FieldMask field_mask = 2; // Field mask to selectively return fields in the response.
+}
+
+// Represents a transaction from any supported blockchain.
+message AnyChainTx {
+  oneof chain {
+    utxorpc.v1alpha.cardano.Tx cardano = 1; // A Cardano transaction.
+  }
+}
+
+// Response containing the transaction associated with the requested hash.
+message GetTxResponse {
```

Comment:
proto lint warnings suggest using names that match the service method. Here we should use `ReadTxResponse`